setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='shifthelper',
-    version='1.1.4',
+    version='1.1.5',
```

Comment:
I would suggest bumping the minor version (New feature release) → 1.2.0

Reply:
okay sure... one sec